src/Engine/GameEvents/MoveEvent.java

```diff
@@ -45,8 +45,7 @@ public void performEvent(MapMaster gameMap)
       GamePath.PathNode endpoint = unitPath.getEnd();
       int fuelBurn = unitPath.getFuelCost(unit, gameMap);
 
-      if( null == gameMap.getLocation(endpoint.x, endpoint.y).getResident() ) // Just avoid triggering a warning.
-        gameMap.moveUnit(unit, endpoint.x, endpoint.y);
+      gameMap.moveUnit(unit, endpoint.x, endpoint.y);
```

Comment:
If this PR is for animating teletile movement faster, why are we also removing a stomp check?

Reply:
It's for visual movement stuff, and I also removed the warning it was trying to avoid.